numba_cuda/numba/cuda/core/ir_utils.py

```diff
@@ -12,7 +12,8 @@
 
 import numba
 from numba.core.extending import _Intrinsic
-from numba.core import types, typing, ir, analysis, postproc, rewrites, config
+from numba.core import types, ir, analysis, postproc, rewrites, config
+from numba.cuda import typing
```

Comment:
Are we in a position where this change could be made across the codebase? Or does this PR change only the instances that are now supported alongside the changes in this PR?